_posts/2018-02-05-14-03-caseInsensitiveAndQualifiedFunctions.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title : "14.3 WebApi URI parser default setting updates: case-insensitive name token and unqualified functions & actions"
```

Comment:
>name token [](start = 74, length = 10)

I wouldn't use "name token"; token is more of an implementation detail.  How about just something like "case-insensitive names and unqualified functions and actions"?  You could shorten to "case-insensitive names and unqualified operations" as operations == functions and actions, but I am fine either way.